packages/sit-onyx/src/components/OnyxNavBar/modules/OnyxNavItem/OnyxNavItem.vue

```diff
@@ -27,7 +27,7 @@ defineSlots<{
     class="onyx-nav-item"
     :active="props.active"
     :href="props.href ?? 'javascript:void(0)'"
-    @click="props.href && emit('click', props.href)"
+    @click="props.href && emit('navItemClick', props.href)"
```

Comment:
Should we remove the click event here so it bubbles the native click event?

Reply:
I remember that we wanted to emit the href on click. This is the reason why I renamed the event